AjaxControlToolkit/AjaxFileUpload/AjaxFileUpload.cs

```diff
@@ -252,6 +252,7 @@ public bool AutoStartUpload
                 return;
                 
             var processor = new UploadRequestProcessor {
+                Control = this,
                 Context = Context,               
```

Comment:
Since we now have a reference to the control, please check if we can remove the `Context` field.

```c#
if(control.Page != null)
    return control.Page.Request.RequestContext.HttpContext
```